consoles/s3270.pm

```diff
@@ -312,12 +313,12 @@ sub nice_3270_status ($self, $status_string) {
 }
 
 sub _connect_3270 ($self, $host) {
-    my $r = $self->send_3270("Connect($host)");
-    confess "connect to host >$host< failed.\n" . join("\n", @$r) if $r->{terminal_status} !~ / C\($host\) /;
+    my $sent = $self->send_3270("Connect($host)");
+    confess "connect to host >$host< failed.\n" . Dump($sent) if $sent->{terminal_status} !~ / C\($host\) /;
```

Comment:
Is this just for temporary debugging or should it stay? Your git commit messages don't really tell a good story there. Please rework them, e.g. split and reword.

Reply:
@perlpunk Can we change this?

Reply:
@perlpunk so do I understand it correctly that we should use `Dump` here in production code and not just for "temporary development purposes"?

Reply:
Yes. This is supposed to print the contents of the variable. Would you suggest something different to do this?
The previous `@$r` could have never worked because it's not an array ref.

Reply:
Ok, fine